.templates/docs.sh.template

```diff
@@ -0,0 +1,98 @@
+{% extends "templates/docs.sh.template" %}
+# Based off NengoDL gpu script. We want to build docs on a machine with FPGAs
+# See https://oncletom.io/2016/travis-ssh-deploy/ for instructions on how the
+# remote server is set up for this script to connect to
+
+{% block install %}
+    # Decrypt private keys in tarball (can't decrypt multiple files currently)
+    # See https://docs.travis-ci.com/user/encrypting-files/
+    exe openssl aes-256-cbc \
+        -K "${encrypted_e86b3ea1b4ca_key:?}" \
+        -iv "${encrypted_e86b3ea1b4ca_iv:?}" \
+        -in .ci/rsa_keys.tar.enc \
+        -out ~/.ssh/rsa_keys.tar -d
+    exe tar -xvf ~/.ssh/rsa_keys.tar -C ~/.ssh
+    exe chmod 600 ~/.ssh/travis-abrgpu_rsa
+    exe chmod 600 ~/.ssh/travis-abrhost_rsa
+
+    # Setup SSH config for abrgpu and abrhost (stored in travis-ci settings)
+    echo -e "$SSH_INFO" >> ~/.ssh/config
+    ssh -o StrictHostKeyChecking=no fpga-ci "echo 'Connected to abrhost'"
+
+    # Setup env on remote server side
+    exe ssh fpga-ci << EOF
+    $(typeset -f exe)  # Send definition of exe function
```

Comment:
The indentation of this block of ssh code matches the block in "script", but does not match the indentation of the blocks in the "after_success" and "after_script" blocks.

Reply:
The indentation was changed for the blocks that call `{{ super() }}`, Since I can't change the indentation of the inherited portion I just matched that, however if I didn't use the `{{ super() }}` block I indented deeper for legibility

Reply:
Hmm. Since the ci scripts are meant to be auto-generated by nengo-bones (and not really for the user to read), i think i'd rather standardize the indentation for readability in the sh template than have 2 levels of indentation. What do you think @drasmuss?

Reply:
I haven't tested, but I'm pretty sure you should be able to change the indentation of the super block with something like `{{ super() | indent(4) }}`

Reply:
Yea that seems to have worked!